app/src/main/java/com/example/plantsapp/data/RoomPlant.kt

```diff
@@ -17,7 +18,7 @@ data class RoomPlant(
             return RoomPlant(
                 plant.name.value,
                 plant.speciesName,
-                plant.plantPicture,
+                plant.plantPicture.toString(),
```

Comment:
and here when it will be a nullable type, change to plant.plantPicture?.toString()